os-config.tpl.yml

```diff
@@ -292,6 +288,7 @@ rancher:
       - /usr/share/ros:/usr/share/ros
       - /var/lib/rancher/cache:/var/lib/rancher/cache
       - /var/lib/rancher/conf:/var/lib/rancher/conf
+      - /var/lib/rancher/docker:/var/lib/rancher/docker
```

Comment:
I think these `/var/lib/rancher/*` are not needed because we bind mount `/var/lib/rancher`.  I think the only reason why we kept these was because some code expected these folders to exist and the bind mount does an implicit `mkdir`.  We should just fix that issue instead.